distribution/lib/Standard/Base/0.0.0-dev/src/Runtime.enso

```diff
@@ -93,7 +94,9 @@ no_inline_with_arg function arg = @Builtin_Method "Runtime.no_inline_with_arg"
    Converts a primitive stack trace element into the regular one.
 wrap_primitive_stack_trace_element el =
     loc = if Polyglot.has_source_location el then Source_Location.Value (Polyglot.get_source_location el) else Nothing
-    name = Polyglot.get_executable_name el
+    name = Panic.catch Type_Error (Polyglot.get_executable_name el) _->
+        # Workaround for a bug where some stack frames' root nodes do not have a name.
+        "<unknown node>"
```

Comment:
This was sometimes failing, around `SQL_Type_Reference.get`,` do_fetch` and `Connection.fetch_columns`. Whenever there were warnings attached to the query result, one entry in the stack trace had `rootNode.getName() == null` and it was crashing the stack trace operation.

I think we better have a stack trace with an `<unknown node>` entry, than prevent adding warnings.

Reply:
Ideally, I'd like to figure out why `rootNode.getName() == null` in the first place, but I could not easily find a minimal example.

@JaroslavTulach any ideas?